datahub/main.py

```diff
@@ -123,3 +124,24 @@ def get_dsr_data(  # type: ignore[misc]
     filtered_data = dt.dsr_data[start : end + 1 if end else end]
 
     return {"data": filtered_data}
+
+
+@app.get("/wesim")
+def get_wesim_data() -> dict[Hashable, Any]:  # type: ignore[misc]
+    """GET method function for getting Wesim data as JSON.
+
+    Returns:
+        A Dict containing the Wesim Dataframes
+    """
+    dt.wesim_data = get_wesim()
```

Comment:
Can we only do this the first time to API is called? So only when `dt.wesim_data` is empty.

```suggestion
    if dt.wesim_data == []:
        dt.wesim_data = get_wesim()
```